src/Integrations/Integrations/CakePHP/V3/CakePHPIntegrationLoader.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace DDTrace\Integrations\CakePHP\V3;
+
+use Cake\Http\ServerRequest;
+use Cake\Routing\Router;
+use DDTrace\HookData;
+use DDTrace\Integrations\CakePHP\CakePHPIntegration;
+use DDTrace\Integrations\Integration;
+use DDTrace\SpanData;
+use DDTrace\Tag;
+use DDTrace\Type;
+use DDTrace\Util\Normalizer;
+
+class CakePHPIntegrationLoader
+{
+    public function load($integration)
+    {
+        $integration->rootSpan = null;
```

Comment:
Can we please move away from caching rootSpan and rather refetch \DDTrace\root_span(), as more and more frameworks are also working under Swoole (where the root span isn't global, but may change with the executing fiber).